azure-kusto-data/source/models.ts

```diff
@@ -70,8 +83,8 @@ export class KustoResultRow {
         return obj;
     }
 
-    toString() {
-        return JSON.stringify(this.toJson());
+    toJsonString() {
```

Comment:
It can break customers... why not leave this name as it is? (also for toObject)

Reply:
This version is going to breaking anyway, so I thought we maybe should have the names be more informative.

ToJson didn't really return "JSON", it returned an object that may have contained in it objects of any type, including date & duration objects, so I think the name isn't correct.

ToString doesn't return a string representation of the class. Usually when I call ToString I expect something human-readable, but what it does is serialize it into JSON.

But if you think keeping compatibility is more important or that the names are good I can roll it back, or maybe leave both options but mark the old ones deprecated.

Reply:
Ok, turns out the convention is to call the method toJSON, and if it exist then JSON.Stringify calls it.

So I reverted ToJsonString to be ToString, and changed toObject (that was toJson) to toJSON.

In the mapping PR I also changed all of the other toJson to toJSON to keep with the convention.

WDYT?

Reply:
Did another change - 
kept the old toJson as deprecated, so we won't have a breaking change.